presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -712,7 +712,7 @@ public static OptionalInt getConcurrentLifespansPerNode(Session session)
             return OptionalInt.empty();
         }
         else {
-            checkArgument(result > 0, "Concurrent lifespans per node must be positive if set to non-zero");
+            checkArgument(result > 0, "Concurrent lifespans per node must be positive if set to non-zero: %s", result);
```

Comment:
That message is confusing.  It should just say `Concurrent lifespans per node is negative: %s`